safe/gui/tools/batch/batch_dialog.py

```diff
@@ -453,6 +615,55 @@ def write_report(self, report, pass_count, fail_count):
         except IOError:
             raise IOError
 
+    def generate_pdf_report(self, impact_function, iface, scenario_name):
+        """Generate map report and impact report from impact function and store
```

Comment:
This also should be one line only for the first line.

Then you should add an empty line and then you can finally add as many lines as you want. (with full stop)